<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="munishmas-mac" name="com.vmetry.testng.Verification" tests="2" failures="2" timestamp="03 Jan 2019 04:06:05 GMT" time="1.008" errors="0">
  <testcase name="validateCricinfo" time="0.020" classname="com.vmetry.testng.Verification">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [http://www.cricinfo.com/] but found [http://www.espncricinfo.com/],
	expected [cricinfo - Cricket Live Scores, Stats, Schedules, Fixtures &amp; News] but found [ESPNcricinfo - Cricket Live Scores, Stats, Schedules, Fixtures &amp; News]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:expected [http://www.cricinfo.com/] but found [http://www.espncricinfo.com/],expected [cricinfo - Cricket Live Scores, Stats, Schedules, Fixtures & News] but found [ESPNcricinfo - Cricket Live Scores, Stats, Schedules, Fixtures & News]at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)at com.vmetry.testng.Verification.validateCricinfo(Verification.java:34)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- validateCricinfo -->
  <testcase name="validateCricBuzz" time="0.988" classname="com.vmetry.testng.Verification">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Cricket Score, Schedule, Latest News, Stats &amp; Videos | cricbuzz.com] but found [Cricket Score, Schedule, Latest News, Stats &amp; Videos | Cricbuzz.com]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:expected [Cricket Score, Schedule, Latest News, Stats & Videos | cricbuzz.com] but found [Cricket Score, Schedule, Latest News, Stats & Videos | Cricbuzz.com]at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)at com.vmetry.testng.Verification.validateCricBuzz(Verification.java:42)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- validateCricBuzz -->
</testsuite> <!-- com.vmetry.testng.Verification -->
